R/dependencies.R

```diff
@@ -1710,7 +1710,7 @@ renv_dependencies_report <- function(errors) {
 renv_dependencies_eval <- function(expr) {
 
   # create environment with small subset of symbols
-  syms <- c("list", "c")
+  syms <- c("list", "c", "!", "+", "-", "*", "/", "||", "&&")
```

Comment:
Do you think there's anything else we should include for a subset of commonly-used things in chunk options?